configuration/configuration.go

```diff
@@ -123,6 +124,22 @@ type Configuration struct {
 	} `yaml:"redis,omitempty"`
 }
 
+// Handlers is for handling error configuration, when some kinds of error happens.
+// Catch the error, and send a warning mail.
+type Handlers struct {
```

Comment:
Given that this is a general configuration file, please provide a better context for the name. Something like `MailHandler` would be preferred.